### PR TITLE
updating readme file

### PR DESCRIPTION
**Pull request by JuMn88**
updating readme file

### CODE REVIEW

**Review by Erdnaxela45 — DISMISSED** [review]

**Review by endricklosange — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -129,5 +82,3 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
```
Comment (LucyMerlier) [inline README.md:83]: copyright 2019 aurelien@wildcodeschool.fr ??? as in Aurélien, my and Sylvain's lead instructor???

File: README.md [hunk 2]
```diff
@@ -94,16 +54,9 @@ Caprover configuration files are :
 * [ESLint](https://eslint.org/)
 * [Sass-Lint](https://github.com/sasstools/sass-lint)
```
Comment (LucyMerlier) [inline README.md:55]: and that is all?

File: README.md [hunk 3]
```diff
@@ -1,88 +1,48 @@
...
-2. Run `composer install`
-3. Run `yarn install`
-4. Run `yarn encore dev` to build assets
-
-### Working
+Please note that this is still a work in progress, and as such, some functionnalities are yet to be implemented, such as :
 
-1. Run `symfony server:start` to launch your local php web server
-2. Run `yarn run dev --watch` to launch your local server for assets
+    [GENERAL] : having a fully responsive website
```
Comment (LucyMerlier) [inline README.md:12]: your website is not yet responsive?

File: README.md [hunk 4]
```diff
@@ -1,88 +1,48 @@
...
+    [GENERAL] : having a complete, clear, and clean README
```
Comment (LucyMerlier) [inline README.md:13]: this is supposed to be the case after this PR

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

**Review by Erdnaxela45 — DISMISSED** [review]

File: README.md [hunk 5]
```diff
@@ -1,88 +1,39 @@
...
-Caprover configuration files are : 
-
-* [captain-definition](https://github.com/WildCodeSchool/sf4-pjt3-starter-kit/blob/master/captain-definition) Caprover entry point
-* [Dockerfile](https://github.com/WildCodeSchool/sf4-pjt3-starter-kit/blob/master/Dockerfile) Web app configuration for Docker container
-* [docker-compose.yml](https://github.com/WildCodeSchool/sf4-pjt3-starter-kit/blob/master/docker-compose.yml) ...not use it's used 😅
-* [docker-entry.sh](https://github.com/WildCodeSchool/sf4-pjt3-starter-kit/blob/master/docker-entry.sh) shell instruction to execute when docker image is built
-* [nginx.conf](https://github.com/WildCodeSchool/sf4-pjt3-starter-kit/blob/master/nginx.conf) Nginx server configuration
-* [php.ini](https://github.com/WildCodeSchool/sf4-pjt3-starter-kit/blob/master/php.ini) Php configuration
+Please note that you may also need to install other packages to fully make everything work together (like php-mysql).
+INSTALLATION
```
Comment (LucyMerlier) [inline README.md:21]: isn't this supposed to be a title? so prefixed with ## like the other titles?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]

**Review by Erdnaxela45 — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**